src/main/webapp/static/js/init.js

```diff
@@ -1,10 +1,31 @@
 (function($){
   $(function(){
-
+    var web3Ready = false;
     $('.button-collapse').sideNav();
     $('#navButton').sideNav();
     
     $('select').material_select();
+    $('#web3').click(function() {
+      if (!web3Ready){
+        if (window.ethereum) {  
```

Comment:
Does `window.ethereum` indicate MetaMask?

Reply:
Yeah, there is more info here: https://docs.metamask.io/guide/provider-migration.html#replacing-window-web3